src/libm-tester/iutsimdmain.c

```diff
@@ -0,0 +1,57 @@
+//          Copyright Naoki Shibata 2010 - 2017.
```

Comment:
I see. Thanks for the explanation. It is easy to miss important bits of information when the patches are big and cover multiple aspects. It would be better if you could submit pull request for small incremental changes.